desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -247,7 +247,12 @@ void TabDiveInformation::updateTextBox(int event) // Either the text box has bee
 				ui->atmPressVal->setText(ui->atmPressVal->text().sprintf("%d",atmpress.mbar));
 				ui->atmPressType->setCurrentIndex(0);    // reset combobox to mbar
 			} else { // i.e. event == COMBO_CHANGED, that is, "m" or "ft" was selected from combobox
-				ui->atmPressVal->clear();   // Clear the text box so that altitude can be typed
+				 // Show estimated altitude
+				bool ok;
+				int i = ui->atmPressVal->text().toInt(&ok,10);
```

Comment:
`i` seems like an odd variable name here - that's typically an index in a loop or something.
And because of this I'm not sure I understand the next part of the code.